src/QsCompiler/CSharpGeneration/Context.fs

```diff
@@ -152,6 +152,7 @@ type CodegenContext =
                 target = AssemblyConstants.HoneywellProcessor
                 || target = AssemblyConstants.IonQProcessor
                 || target = AssemblyConstants.QCIProcessor
+                || target = "AzureSimulator" // ToDo: We need to have an assembly constant for this.
```

Comment:
```suggestion
                || target = "MicrosoftSimulator" // ToDo: We need to have an assembly constant for this.
```

Reply:
I'm not sure this is right. While debugging the name that came up was "AzureSimulator", not "MicrosoftSimulator".

Reply:
This is correct.